src/main/java/org/apache/pirk/responder/wideskies/spark/ComputeResponse.java

```diff
@@ -202,7 +203,7 @@ private void setup() throws Exception
     numColMultPartitions = Integer.parseInt(SystemConfiguration.getProperty("pir.numColMultPartitions", numDataPartsString));
 
     // Whether or not we are performing a reduceByKey or a groupByKey->reduce for column multiplication
-    colMultReduceByKey = SystemConfiguration.getProperty("pir.colMultReduceByKey").equals("true");
+    colMultReduceByKey = SystemConfiguration.getProperty("pir.colMultReduceByKey", "false").equals("true");
```

Comment:
In scalability testing, reduceByKey for column multiplication proved far less efficient than groupByKey followed by a map functionality (hence the 'false' default value).